src/Compilers/Core/Portable/Symbols/SymbolEqualityComparer.cs

```diff
@@ -24,6 +24,8 @@ public sealed class SymbolEqualityComparer : IEqualityComparer<ISymbol>
         // Internal only comparisons:
         internal readonly static SymbolEqualityComparer ConsiderEverything = new SymbolEqualityComparer(TypeCompareKind.ConsiderEverything);
 
+        internal readonly static SymbolEqualityComparer IgnoreNullableAndTupleNames = new SymbolEqualityComparer(TypeCompareKind.AllIgnoreOptions | TypeCompareKind.IgnoreTupleNames);
```

Comment:
>TypeCompareKind.AllIgnoreOptions | TypeCompareKind.IgnoreTupleNames [](start = 113, length = 67)

It looks like the set of flags doesn't match the name of the field. Also, ```TypeCompareKind.AllIgnoreOptions``` already ignores everything, so I think adding ```TypeCompareKind.IgnoreTupleNames``` doesn't change anything.

Reply:
It was intended to be AllNullable, must have mistyped. I'll remove this declaration entirely.

---
In reply to: [327333301](https://github.com/dotnet/roslyn/pull/38769#discussion_r327333301) [](ancestors = 327333301)